#56: Support for default locale in realm update

aberg09 opened the pull request:
There are only the changes Realm.kt and UpdateRealmAction.kt sofar.

I have 1000+files in the docs and docsbuild directories that got changed due to the documentation autogeneration. Those should probably not be oart of the commit. Is there something wrong with the gitingore file here? 

Tests:
I confirmed it works manually.
I was not sure whether (or where) to add automated tests. Please provide some guidance here.

klg71 commented:
Hey aberg thanks for your pull request <3
I have to commit the docs because they are served through github. I will just ignore them while reviewing your MR :)
There are automated integration tests for each Action. In your case you could add it to the "UpdateRealmIntegTest"-Test.
The tests will start a local keycloak to execute if you have any trouble with this, please let me know.
Best Regards :)

klg71 commented on keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/model/Realm.kt:
```diff
@@ -84,6 +84,7 @@ data class Realm(
     val adminEventsDetailsEnabled: Boolean,
     val internationalizationEnabled: Boolean,
     val supportedLocales: List<String>,
+    val defaultLocale: String = "English",
```
Are we sure `English` is a good default here? If you set it through the UI:
![image](https://user-images.githubusercontent.com/7859791/221166844-8e4005e7-91fa-4542-a5fd-eb1e247cf10a.png)
The value is `en`. I don't know if English also works fine.

klg71 commented:
Review completed. Code looks good just one minor issue :) Thanks for your contribution.

aberg09 commented:
Hi
thank you for the project to begin with :)

I have changed the defaul to 'en' as you suggested and tested that it works.
I have also added the test in UpdateRealmIntegTest and the docs.

Please let me know, if something else is missing.
